omod/src/main/java/org/openmrs/module/spa/servlet/SpaResourcesServlet.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.spa.servlet;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.spa.utils.SpaModuleUtils;
+import org.openmrs.util.OpenmrsUtil;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * This is a servlet class that handles all requests for the frontend static resources
+ * All requests with /frontend are handled by the class
+ */
+public class SpaResourcesServlet extends HttpServlet {
+
+	private static final long serialVersionUID = 1239820102030344L;
```

Comment:
This is the UID for the Module Resources Servlet. I think you're supposed to define a new UID, since this is a different class.

Reply:
Fixed